evaluator.py

```diff
@@ -0,0 +1,40 @@
+# Python Library import
+import time
+
+# Project imports
+import util
+
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(__file__)), 'aggregator'))
+
+class Evaluator:
+
+  def init(self, start_time, end_time):
+    self.start_time = start_time
+    self.end_time = end_time
+    self.aggregator = Aggregator()
+
+  def evaluate(self):
+    # collect server logs
+    logs = self.aggregator.get_log_entries(self.start_time, self.end_time)
```

Comment:
sorry about not updating with your code refactoring. I fixed it by adding a function in aggregator that can retrieve logs related to replication.